### ISSUE TITLE
Steam thermal limit not recovered

### ISSUE THREAD

**Issue by timlichtenberg** [FormingWorlds/AEOLUS#29]
@arejaygraham reported this a while ago; the latest code version does not reproduce the pure steam thermal limit anymore. With a surface temperature of 800 K and 260 bar H2O I get an OLR of 469.422 W/m^2 (no scattering, no tropopause, SOCRATES 2211), which is too high (it should be ~280 W/m^2). 

@arejaygraham reported that using a spectral file from the ROCKE-3D website lowered the limit to ~290 W/m^2, so it could be a spectral file issue.

**Comment by timlichtenberg** [comment]
- [ ] Check with different spectral file
- [ ] Check with different SOCRATES version

**Comment by hinnes97** [comment]
I have tested this with the (probably old) version of SOCRATES that I was using for my Hycean world work. When I use the spectral file `sp_b318_HITRAN_a16`, I get 465 W/m^2 like you do. Using the spectral files I generated, I get 280 W/m^2, so the error is in the spectral file.

Testing with and without the `-u` flag in `Cl_run_cdf` (turning on/off the continuua), using `sp_b318_HITRAN_a16` I get an identical value for the OLR (whereas with my spectral file, turning off the continuum the OLR jumps to 490 W/m^2).

TL;DR for some reason there is no H2O continuum in the spectral file

**Comment by hinnes97** [comment]
Confirmed, there is no H2O-H2O continuum in the spectral file. H2O-H2O has index "1", and is not on the list inside the spectral file for any band
<img width="613" alt="Screenshot 2023-06-29 at 12 28 48" src="https://github.com/FormingWorlds/AEOLUS/assets/56552136/ee35ebaa-3781-416f-a7ac-1ed2c3fd74f0">

**Comment by Ryan-Boukrouche** [comment]
I used socrates 2306 to generate a new spectral file for water using HITRAN and MT_CKD data. This time the continuum did go in and the radiation limit is acceptable, about 277-278 W/m2. A lingering issue is that the opacity seems overestimated according to the attached figure. The flux should increase sooner, around 2000 K.
[Total_flux_up_limit.pdf](https://github.com/FormingWorlds/AEOLUS/files/11929368/Total_flux_up_limit.pdf)

**Comment by timlichtenberg** [comment]
- [ ] Generate new general spectral file(s) that combines all opacity sources for redox-agnostic simulations.
- [ ] Generate one for H2/H2O only (sub-Neptunes)

**Comment by timlichtenberg** [comment]
As discussed in Discord, looks working nicely! :-) For all others: what the script needs as of today is SOCRATES with a version smaller than 2211 and HITRAN data files according to this folder arrangement: https://www.dropbox.com/sh/hz5b4d2e9qfnnuh/AADUcjxTgDNfW41SV9YWZ36_a?dl=0. We'll be generating now a couple of spectral files for various arrangement in the next few weeks and deposit them somehow publicly; then generalise the spectral file options in the main run scripts of PROTEUS and AEOLUS.

If there are specific wishes for spectral files, shoot. ;-)

**Comment by Ryan-Boukrouche** [comment]
I did use Socrates 2306, since the hitran and mt_ckd data is the same, maybe there is still a bug in the new socrates version. That, or something has been fixed... But the comparison with Hamano etc. suggests the former

**Comment by nichollsh** [comment]
I've also tried this with SOCRATES 2211 and they don't exactly produce the same results. The spectral files differ in that the 2211 version uses more k-terms. Maybe that explains the difference, because it's pretty minimal. SOCRATES versions used to run the code for a given spectral file were consistent with the versions used to make the file itself.

See the plot below. Blue: SOCRATES 2306. Orange: Hamano+15. Yellow: SOCRATES 2211.

I think the results are similar enough that we don't need to worry too much, but it's something to be aware of.

<img width="803" alt="SCR-20230711-pdn" src="https://github.com/FormingWorlds/AEOLUS/assets/32022679/a0f57d03-7ade-4076-a11f-5c5405534fdf">
